charts/rollups-node/README.md

```diff
@@ -2,7 +2,7 @@
 
 # Package for Cartesi Rollups Nodes
 
-![Version: 1.3.0-0](https://img.shields.io/badge/Version-1.3.0--0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 1.3.0-rc.1](https://img.shields.io/badge/Version-1.3.0--rc.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
```

Comment:
This will show the current PR action status. So, In My Idea, Every PR should have it's own README based on the charts and versions. So, when we merge this PR, we should regenerate the README based on final PR.

Reply:
![image](https://github.com/cartesi/helm-charts/assets/51019382/8e355337-5c49-4cc7-aeed-46db8674d38d)

This picture is related to README of this PR.